### DISCUSSION TITLE
UI issues with the positioning of the carousel arrows

### DISCUSSION THREAD

**Discussion by jonnymarshall** [lukevella/rallly#661] [General]
Hi, first time here but been using Rallly for a couple of months and it's been generally fantastic.

One significant problem however that I've been having recently, is since the pagination arrows which move the time slots carousel to the next batch of options have moved (I believe they used to be to the left and right of the timeslots themselves), people are very often missing that there's more options than the initial batch which is immediately visible on the page.

I'm talking about these arrows:
<img width="1236" alt="Screenshot 2023-04-14 at 18 18 48" src="https://user-images.githubusercontent.com/42963069/232173536-760f20a2-3190-4993-9224-a1d25aa2481f.png">

I'm no UI expert and generally speaking I think Rallly has excellent UI, so I won't be specific as to where I think those arrows should be, however the previous placement seemed to work great and no one had any issues. Now when I have more options than fit in the initial carousel slide, there's a very high chance of people not knowing that more options exist, as the placement of those arrows is now very easy to miss, and there's no obvious visual cue that there's more options. So I think either reverting to their old placement in the last version, or finding a new placement or some  way of making them more visible, is needed.

Side note: Thanks to all the devs working on this. It's a great tool and I'm looking forward to seeing some of the new features suggested in the issues.

**Comment by lukevella** [comment]
Hey @jonnymarshall, thanks for the feedback. I agree with you that the current positioning is not ideal. It seems that anywhere we move these there is always some issue which is why in the next major release which will hopefully start being rolled out in a few weeks we'll be taking a different approach that should address your concern. Keep an eye out for the v3 announcement, it'd be great to get your feedback once it's available.

**Reply by jonnymarshall** [reply]
Thanks, will be happy to provide feedback once v3 is out. Just spitballing here but another option (perhaps a temporary patch) might be to have a conditional confirmation dialog if the individual has not clicked the button to show the full range of options, suggesting that they go back and check out the full range of options before submitting. If they do click the arrow then this dialogue would not display.

Just a thought. Either way, looking forward what improvements you've got in store for v3.

**Reply by lukevella** [reply]
Makes sense. I'll try and see if I can work that in but as I'm the only maintainer at the moment it may be tough to dedicate the necessary time for that as my current focus is v3. Thanks for the feedback.
